src/sentry/web/frontend/accounts.py

```diff
@@ -463,71 +469,80 @@ def show_emails(request):
 
         return HttpResponseRedirect(request.path)
 
-    if email_form.is_valid():
-        old_email = user.email
+    if 'primary' in request.POST:
+        new_primary = request.POST.get('new_primary_email').lower()
```

Comment:
you can just do `request.POST['new_primary_email']` since if `.get` returns None, you'll still have an error on `.lower`